RadixWallet/Features/DisplayEntitiesControlledByMnemonic/DisplayEntitiesControlledByMnemonic+Reducer.swift

```diff
@@ -9,22 +9,63 @@ public struct DisplayEntitiesControlledByMnemonic: Sendable, FeatureReducer {
 
 		public var deviceFactorSource: DeviceFactorSource { accountsForDeviceFactorSource.deviceFactorSource }
 
-		public let accountsForDeviceFactorSource: EntitiesControlledByFactorSource
-		public let displayRevealMnemonicLink: Bool
+		public var accountsForDeviceFactorSource: EntitiesControlledByFactorSource
+		public var displayRevealMnemonicLink: Bool {
+			switch mode {
+			case .mnemonicCanBeDisplayed: true
+			case .mnemonicNeedsImport: false
+			case .displayAccountListOnly: false
+			}
+		}
+
+		public var mnemonicNeedsImport: Bool {
+			switch mode {
+			case .mnemonicCanBeDisplayed: false
+			case .mnemonicNeedsImport: true
+			case .displayAccountListOnly: false
+			}
+		}
```

Comment:
I would just skip these and have a switch statement in body. If we need to check in other places then `mode == .mnemonicNeedsImport` is at least as clear as the boolean accessor I think.